src/config/config.php

```diff
@@ -210,7 +210,7 @@
         //'az-Cyrl'     => ['name' => 'Azerbaijani (Cyrillic)', 'script' => 'Cyrl', 'native' => 'Азәрбајҹан', 'regional' => 'uz_UZ'],
         //'ab'          => ['name' => 'Abkhazian',              'script' => 'Cyrl', 'native' => 'Аҧсуа', 'regional' => ''],
         //'os'          => ['name' => 'Ossetic',                'script' => 'Cyrl', 'native' => 'Ирон', 'regional' => 'os_RU'],
-        //'ky'          => ['name' => 'Kyrgyz',                 'script' => 'Cyrl', 'native' => 'Кыргыз', 'regional' => 'ky_KG'],
+        //'kg'          => ['name' => 'Kyrgyzstan',                 'script' => 'Cyrl', 'native' => 'Кыргызча', 'regional' => 'kgz_KG'],
```

Comment:
1) I think here should be Language name, not Country Name.
Language name `'name' => 'Kyrgyz'`, not `'name' => 'Kyrgyzstan'`

2) Based on list of locales it should be `'regional' => 'kk_KZ'`
```
bash$ locale -a
...
kk_KZ
kk_KZ.PT154
kk_KZ.UTF-8
...
```
3. And key of this array element should be 'ky', because of key for the Kyrgyz Language in ISO 639-1 is `ky`